schema/rdl_schema_0.1.json

```diff
@@ -949,6 +918,61 @@
 				"TCY"
 			]
 		},
+		"Resource": {
+			"title": "Resource",
+			"type": "object",
+			"description": "Information about an individual version of the dataset.",
+			"required": [
+				"id",
+				"title",
+				"description",
+				"media_type",
+				"url"
+			],
+			"properties": {
+				"id": {
+					"title": "Resource identifier",
+					"type": "string",
+					"description": "An identifier for this resource. This should be internally unique."
+				},
+				"title": {
+					"title": "Resource title",
+					"type": "string",
+					"description": "A title for this resource."
+				},
+				"description": {
+					"title": "Resource description",
+					"type": "string",
+					"description":"Additional description of the resource not already covered by th eother resource fields." 
+				},
+				"media_type": {
```

Comment:
Per https://github.com/GFDRR/rdl-standard/issues/54#issuecomment-1566310558, we should also have a `format` field.

Reply:
added